match/spec/storage/git_storage_spec.rb

```diff
@@ -43,6 +43,46 @@
         expect(File.exist?(File.join(storage.working_directory, 'README.md'))).to eq(false) # because the README is being added when committing the changes
       end
 
+      it "directly clones a single branch wtih shallow clone" do
```

Comment:
Typo:

```suggestion
      it "directly clones a single branch with shallow clone" do
```

Despite the typo, this rspec structure is a bit weird to me. I'd normally expect something like

```ruby
describe "#download"
  # Insert some set up code like lacostej mentioned

  describe "when using shallow_clone and clone_branch_directly" do
    it "clones a single branch" do
      …
    end
  end
end
```

What do you think?

Reply:
This is not critical though, the rest of the file is structured like this already so I guess it's at least consistent 😅 if you don't feel strongly about fixing this it's ok, we can simply fix the typo then